src/gpg/contact/list.py

```diff
@@ -0,0 +1,38 @@
+import gpg
+import sys
+import os
+from fn.print_key import print_key
+
+
+def ls(contact_list):
+    """
+    list the contacts in contact list
+    """
+    try:
+        c = gpg.Context()
+
+        key_set = set({})
+
+        for contact in contact_list:
+            key_set.update(set(c.keylist(contact)))
```

Comment:
I think you might want to use the `set.add()` method or are we missing some use-case?

Reply:
A contact (contact string) can have multiple matches. This is intended to list all of them.

Reply:
Set is required as in case of multiple arguments, like `egpg contact ls divesh $MYKEYID` i want to list each contact only once.

Reply:
if `c.keylist` returns a list, you don't need to convert it into a set is what I'm inferring, it will do that on its own afaik.

Reply:
It returns a generator but yup it will do the job I guess.